pkgs/by-name/mu/museum/package.nix

```diff
@@ -5,23 +5,24 @@
   libsodium,
   buildGoModule,
   nix-update-script,
+  nixosTests,
 }:
 
-buildGoModule rec {
+buildGoModule (finalAttrs: {
   pname = "museum";
```

Comment:
I would propose to renames this to ente-museum as this is a rather generic name.